third_party/move/move-binary-format/src/file_format_common.rs

```diff
@@ -222,7 +222,7 @@ pub enum SerializedFunctionAttribute {
 #[rustfmt::skip]
 #[allow(non_camel_case_types)]
 #[repr(u8)]
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
@wrwg , @vgao1996 

are these changes fine (I need to have these in the key of the BTreeMap), or is there a different suggestion here?

Reply:
Should be fine